src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -5,67 +5,76 @@
       "Size": 3032
     },
     "assemblies/Java.Interop.dll": {
-      "Size": 60736
+      "Size": 60761
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 152435
+      "Size": 156719
     },
     "assemblies/rc.bin": {
       "Size": 1083
     },
     "assemblies/System.Collections.Concurrent.dll": {
-      "Size": 8757
+      "Size": 9171
     },
     "assemblies/System.Collections.dll": {
-      "Size": 4196
+      "Size": 4282
     },
     "assemblies/System.Collections.NonGeneric.dll": {
-      "Size": 6232
+      "Size": 6236
     },
     "assemblies/System.Console.dll": {
-      "Size": 6502
+      "Size": 6506
+    },
+    "assemblies/System.Formats.Asn1.dll": {
+      "Size": 17426
     },
     "assemblies/System.IO.Compression.dll": {
-      "Size": 16259
+      "Size": 16266
     },
     "assemblies/System.Linq.dll": {
-      "Size": 10183
+      "Size": 10316
     },
     "assemblies/System.Net.Http.dll": {
-      "Size": 65104
+      "Size": 75341
+    },
+    "assemblies/System.Net.NameResolution.dll": {
+      "Size": 12592
     },
     "assemblies/System.Net.Primitives.dll": {
-      "Size": 21115
+      "Size": 32401
     },
     "assemblies/System.Net.Requests.dll": {
-      "Size": 3515
+      "Size": 3519
     },
     "assemblies/System.Private.CoreLib.dll": {
-      "Size": 613751
+      "Size": 621008
     },
     "assemblies/System.Private.Uri.dll": {
-      "Size": 37595
+      "Size": 37777
     },
     "assemblies/System.Private.Xml.dll": {
-      "Size": 140804
+      "Size": 140803
     },
     "assemblies/System.Runtime.dll": {
-      "Size": 2405
+      "Size": 2410
+    },
+    "assemblies/System.Runtime.Numerics.dll": {
+      "Size": 23826
```

Comment:
When #7059 is merged it will fix the problem for most use cases. It will still add bloat whenever customers use `Credentials` or `Proxy` even when they don't need NT auth.

Reply:
@simonrozsival: #7059 has been merged.  Please rebase this PR and fix the merge conflicts.